packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -57,6 +57,31 @@ export enum FlushMode {
     TurnBased,
 }
 
+/**
+ * This tells the visibility state of a Fluid object. It basically tracks whether the object is not visible, visible
+ * locally within the container only or visible globally to all clients.
+ */
+export enum VisibilityState {
```

Comment:
enums create all sorts of problems, could we use and object instead:  [VisibilityState](https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums)

Reply:
We are using string unions in couple other places; I think that results in more compact code (compare to enums like below), and are better for telemetry readability.

Reply:
Changed it to an object.